src/verify/mod.rs

```diff
@@ -27,6 +27,15 @@ pub trait Credential {
     fn parse_claims(claims: ClaimsSet) -> Result<HashMap<String, ClaimValue>>;
 }
 
+pub fn retrieve_entry_from_status_list(status_list: String, idx: usize) -> Result<u8, DecodeError> {
+    let status_list: JsonStatusList =
+        serde_json::from_str(status_list.as_str())
+            .map_err(|e| DecodeError::Zlib(e.into()))?;
```

Comment:
This error variant is for ZLib compression. I don't think there's any reason to use a specific error type for this function, so you can just use `anyhow::Error`.